webapp/components/Sidebar.js

```diff
@@ -22,16 +22,28 @@ class Sidebar extends PureComponent {
       customSidebarFooter: PropTypes.node,
       location: PropTypes.shape({
         pathname: PropTypes.string
-      })
+      }),
+      match: PropTypes.shape({
+        isExact: PropTypes.bool,
+        path: PropTypes.string,
+        url: PropTypes.string,
+        params: PropTypes.object
+      }).isRequired
     };
   }
 
   renderLogo() {
-    const { theme } = this.props;
+    const {
+      theme,
+      match: { url }
+    } = this.props;
     return (
       <Link to="/">
         <div className={theme.sidebar.logoContainer}>
-          <img src={theme.logoUrl} className={theme.sidebar.logoImg} />
+          <img
+            src={`${url}${theme.logoUrl}`}
```

Comment:
Will the `${url}${theme.logoUrl}` always render properly with the `url` having a `/` suffix?

Reply:
It'll depend on what they pass in. But yeah, this looks like it breaks if you use an external resource (as the bmenace skin does in the genesis theme). I'll try and fix this.